openedx/frontend-app-authoring
# feat: add tags to collections [FC-0062]

## Description

This PR adds the `ContentTagsDrawer` to the Collection Info sidebar.

- Which edX user roles will this change impact? "Course Author"

![image](https://github.com/user-attachments/assets/eecb90eb-8254-4767-a6e3-a6761de41446)

## Supporting information

- Internal ticket: [FAL-3887](https://tasks.opencraft.com/browse/FAL-3887)

## Testing instructions

- Run this script to create taxonomies: https://github.com/open-craft/taxonomy-sample-data
- Create a new library or use an existing one.
- Create a new collection.
- Open the Collection Info by clicking on the collection card.
- Add tags and verify that the tags count in the collection card is updated
- Play with the content tags drawer (add, delete)
